.github/workflows/release.yml

```diff
@@ -13,6 +13,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
+    - uses: everlytic/branch-merge@1.1.2
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        source_ref: "develop"
+        target_branch: "latest"
+        commit_message_template: 'RELEASE: :shipit: :boom: :tada: Merged {source_ref} into target {target_branch}'
```

Comment:
We shouldn't do that if the release was not tagged on develop. Namely I'm thinking of pre-releases.

One option is to check if `rc` is in the tag?

Reply:
Ooh yeah that makes sense